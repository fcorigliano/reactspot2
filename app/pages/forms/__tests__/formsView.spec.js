const React = require('react');
const FormsView = require('../view');
const { render, screen, fireEvent } = require('@testing-library/react');

describe('La view de la page Forms', () => {
    let component;
    const i18n = { gettext: text => text };

    beforeEach(() => {
        component = render(<FormsView i18n={i18n}/>);
    });

    it('1) Renderiza', () => {
        const { asFragment } = component;
        expect(asFragment()).toMatchSnapshot();
    });
});