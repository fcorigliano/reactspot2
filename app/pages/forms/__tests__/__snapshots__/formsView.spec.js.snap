// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`La view de la page Forms 1) Renderiza 1`] = `
<DocumentFragment>
  <div
    class="demo"
  >
    <form
      id="precio"
    >
      <label>
        Rango de Precio
        <input
          name="min"
          placeholder="Min"
          type="number"
          value=""
        />
        <input
          name="max"
          placeholder="Max"
          type="number"
          value=""
        />
      </label>
      <button
        type="submit"
      >
        Agregar rango de precio
      </button>
    </form>
    <form
      id="categoria"
    >
      <label>
        <input
          name="category"
          placeholder="Categoria"
          type="text"
          value=""
        />
      </label>
      <button
        type="submit"
      >
        Agregar filtro por categor√≠a
      </button>
    </form>
    <h2>
      Filtros
    </h2>
    <ul />
    <h2>
      Feedback
    </h2>
    <form>
      <input
        name="name"
        placeholder="Ingresa tu nombre..."
        type="text"
      />
      <textarea
        name="comment"
        placeholder="Dejanos un comentario!"
      />
      <button
        type="submit"
      >
        Dar feedback
      </button>
    </form>
    <ol />
  </div>
</DocumentFragment>
`;
